js/src/html/beautifier.js

```diff
@@ -376,7 +376,10 @@ Beautifier.prototype._handle_tag_close = function(printer, raw_token, last_tag_t
   } else {
     if (last_tag_token.tag_start_char === '<') {
       printer.set_space_before_token(raw_token.text[0] === '/', true); // space before />, no space before >
-      if (this._is_wrap_attributes_force_expand_multiline && last_tag_token.has_wrapped_attrs) {
+      // Add newline before tag closing bracket:
+      // 1. add newline only if 'force_expand_multiline' or 'closing-bracket-newline' is specified
+      // 2. add newline only if tag has wrapped_attrs
+      if ((this._is_wrap_attributes_force_expand_multiline || this._options.closing_bracket_newline) && last_tag_token.has_wrapped_attrs) {
```

Comment:
This makes the behavior of this new option the same as `force_expand_multiline` ...  If that is the case you don't need the new option.  I think what you want is 
```suggestion
      if ((this._is_wrap_attributes_force_expand_multiline && last_tag_token.has_wrapped_attrs)  || this._options.closing_bracket_newline) {
```

But that might not be it either...

Reply:
> This makes the behavior of this new option the same as `force_expand_multiline` ... If that is the case you don't need the new option.

Yes, it is. This new option should work only with a tag which has wrap attributes, it should not work when tag has no attributes.